src/app/app-routing.module.ts

```diff
@@ -10,14 +11,14 @@ const routes: Routes = [
     children: [
       {
         path: '',
-        data: { title: 'Home' },
-        canActivate: [MsalGuard],
+        data: { title: 'Home', roles: ['CVSFullAccess', 'TechRecord.Read'] },
```

Comment:
Could we have `VTMAdmin` added to this list wherever `CVSFullAccess` is mentioned as well please?

Reply:
Hi, yes, I have currently left this off so I can test this before platform actually do there bit. When this moves to develop, I will add in this everywhere too, but leaving it off for now allows me to use a CVSFullAccess to see it works, then use a VTMAdmin to prove it does not

Reply:
OK, but if this is merged into `develop` then most if not all of the devs will be locked out!

Could we merge this into a feature branch instead, verify the change there, then add the `VTMAdmin` role before merging into `develop` please?

Reply:
Sorry - I see you are intending to merge this into `VTMDEV-1` feature env, thank you!